### PR TITLE
Add endian.h shim for macOS build.

### PR DESCRIPTION
**Pull request by npes-95**
Added a shim so the code will build on macOS.

### CODE REVIEW

File: helpers/CMakeLists.txt [hunk 1]
```diff
@@ -6,3 +6,7 @@ include_directories(..)
 file(GLOB SOURCES "*.c")
 
 add_library(helpers STATIC ${SOURCES})
+
+if (${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
+    include_directories(helpers/macos)
```
Comment (pcppcp) [inline helpers/CMakeLists.txt]: could you rename `helpers/macOS/endian.h` -> `helpers/macos/endian.h`?

Comment (npes-95) [reply helpers/CMakeLists.txt]: yep just renamed

Comment (npes-95) [reply helpers/CMakeLists.txt]: the cmake build throws up the same error as mentioned in issue #60, so the cmake build isn't working

**Comment by pcppcp** [comment]
btw you don't have to re-create changes pushed to the `master`. Simply rebase:
```
(your_branch) git remote add upstream git@github.com:AnyLedger/anyledger-wallet.git
(your_branch) git checkout master
(master) git pull upstream master
.... master updates ...
(master) git checkout your_branch
(your_branch) git rebase master
```

File: helpers/CMakeLists.txt (hunk 1 above)
Comment (npes-95) [reply helpers/CMakeLists.txt]: okay solved that issue, just have to change the cmake to `include_directories(macos) `

**Pull request closed without merging**